shared/pingpong/pingpong.go

```diff
@@ -384,6 +384,11 @@ func sendFromTo(
 	amt := cfg.MaxAmt
 	fee := cfg.MaxFee
 
+	assetsByCreator := make(map[string][]*v1.AssetParams)
```

Comment:
nit: size the map according to the size of the cinfo.AssetParams

Reply:
the problem is exact size is not known, each address creates multiple assets. So let it grow.